raw_data.php

```diff
@@ -72,8 +78,8 @@
             }
         }
 
-        if (isset($_GET['eids'])) {
-            $eids = explode(",", $_GET['eids']);
+        if (isset($_POST['eids'])) {
```

Comment:
Consider replacing `isset` with `empty` (and I'll stop repeating myself after this one :P).